job-server/src/spark.jobserver/WebApi.scala

```diff
@@ -531,6 +536,36 @@ class WebApi(system: ActorSystem,
   override def timeoutRoute: Route =
     complete(500, errMap("Request timed out. Try using the /jobs/<jobID>, /jobs APIs to get status/results"))
 
+  def userNamePrefix(authInfo: AuthInfo) : String = {
```

Comment:
One last comment, can you please document this explaining what we talked in the issue (maybe add a link to the issue)